stardis/model/geometry/radial1d.py

```diff
@@ -15,6 +17,8 @@ class Radial1DGeometry:
         distance to the next depth point
     """
 
+    hdf_properties = ["r"]
```

Comment:
This too now doesn't need to be added.

Reply:
Addressed in https://github.com/tardis-sn/stardis/pull/213/commits/140f06b511f1555f4c6f96f469126b24ff35a344